### PR TITLE
As/automation v2 merged

### PR DESCRIPTION
**Pull request by adamskrodzki**
merge

### CODE REVIEW

File: test/dummy-aave-trigger.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        await hre.ethers.provider.send('evm_revert', [snapshotId])
+    })
+
+    it('should be able to add Trigger', async () => {
+        const dataToSupply = AutomationBotInstance.interface.encodeFunctionData('addTriggers', [
+            TriggerGroupType.SingleTrigger,
+            [true],
+            [0],
+            [triggerData],
+            [TriggerType.SimpleAAVESell],
```
Comment (halaprix) [inline test/dummy-aave-trigger.ts:119]: there's not `SimpleAAVESell` in `TriggerType`

File: test/dummy-aave-trigger.ts [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                encodedData,
+                {
+                    gasLimit: 10000000,
+                    value: hre.ethers.BigNumber.from(10).mul(hre.ethers.BigNumber.from(10).pow(18)),
+                },
+            )
+        ).wait()
+
+        const args = [DPMAccount.address, TriggerType.SimpleAAVESell, '1000000', 1800, DPMAccount.address]
+        const types = getDefinitionForCommandType(CommandContractType.SimpleAAVESellCommand)
```
Comment (halaprix) [inline test/dummy-aave-trigger.ts:101]: there's no `CommandContractType.SimpleAAVESellCommand`

File: test/dummy-aave-trigger.ts [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+    beforeEach(async () => {
+        snapshotId = await hre.ethers.provider.send('evm_snapshot', [])
+    })
+
+    afterEach(async () => {
+        await hre.ethers.provider.send('evm_revert', [snapshotId])
+    })
+
+    it('should be able to add Trigger', async () => {
```
Comment (halaprix) [inline test/dummy-aave-trigger.ts:113]: that fails

File: test/dummy-aave-trigger.ts (hunk 1 above)
Comment (adamskrodzki) [reply test/dummy-aave-trigger.ts:119]: "@oasisdex/automation": "1.2.7-alpha.2"

this is package.json

**Review by halaprix — APPROVED** [review]

**Pull request merged**